pages/tools/fileconverter.vue

```diff
@@ -241,70 +234,44 @@ export default {
           href: 'https://github.com/Geode-solutions/OpenGeode',
         },
       ],
+      loading: false,
+      multiple: false,
+      versions: [],
+      multiple: false,
+      objects: [],
+      success: false,
     }
   },
   computed: {
-    path() {
-      return this.API + '/' + this.ID
-    },
+    ...mapState(['ID', 'cloudRunning']),
   },
   created() {
-    this.CheckID() // Lauches the AWS Lambda function
+    if (process.client) {
+      this.CheckID()
```

Comment:
Should we rename this with something like `connect` or `AWSConnect`or `createConnexion` or ... ?

Reply:
I like "createConnexion"